parm/config/config.base.emc.dyn

```diff
@@ -227,11 +233,77 @@ export lobsdiag_forenkf=".true."
 export DO_GLDAS="YES"
 export gldas_cyc=00
 
-# run wave component
-export DO_WAVE="NO"
+# APP settings
+export APP="ATM"
```

Comment:
How does APP get set? Is this where a user would modified it?

Reply:
A user could modify here and/or this could be added to the variables in config.base that the setup script arguments override. The default could be "ATM" but a user provided flag (e.g. --app) given to setup_expt*py could override. Thoughts?

Reply:
Let's just get it in for now and worry about making it more user-friendly later.